buildpack.toml

```diff
@@ -11,5 +11,14 @@ keywords = ["procfile", "processes"]
 [[stacks]]
 id = "*"
 
+[[stacks]]
```

Comment:
Is it worth adding a comment saying why these are added, and that they can be removed once users have newer Pack installed (at some point we'll no doubt rev the buildpack API version to a point where newer pack is needed to understand it anyway, at which point we can know everyone must have upgraded).

Reply:
Done here: https://github.com/heroku/procfile-cnb/pull/55/commits/33c75f01ed7f36994b661a07cd79b191c94a9e27